appvm-scripts/etc/profile.d/qubes-gui.sh

```diff
@@ -1,4 +1,6 @@
-export DISPLAY=:0 _JAVA_AWT_WM_NONREPARENTING=1
+if ! ([ -n "$SSH_CLIENT" ] || [ -n "$SSH_TTY" ] ]); then
+  export DISPLAY=:0 _JAVA_AWT_WM_NONREPARENTING=1
+fi
```

Comment:
First, this should apply only to `DISPLAY`, not the other variable.
But second, having it specifically for SSH isn't very elegant. Maybe better simply check if it's set already and not override it then? This can be done simpler:
```
: ${DISPLAY:=:0}
export DISPLAY
```